scripts/zones/Waughroon_Shrine/mobs/Princess_Jelly.lua

```diff
@@ -0,0 +1,185 @@
+-----------------------------------
+-- Area: Waughroon Shrine
+--  MOB: princess_jelly
+-- Author: Setzor (ported to LSB by dallano)
+-----------------------------------
+require("scripts/globals/pathfind")
+require("scripts/globals/utils")
+-----------------------------------
+local princesses =
+{
```

Comment:
While not necessary, I think for readability's sake, it might be beneficial to explicitly key these values for the BCNM ID